src/open_inwoner/plans/models.py

```diff
@@ -158,3 +155,12 @@ def get_other_users(self, user=None):
         from open_inwoner.accounts.models import User
 
         return User.objects.filter(id__in=user_ids)
+
+    def get_status(self):
```

Comment:
I think the multi-line style is more readable as you don't need to read the whole thing, scales better when modified and has better diffs.

```
if self.end_date > date.today():
    return _("Open")
else:
    return _("Afgerond")  
```